charts/artifacts/templates/deployment.yaml

```
@@ -46,20 +46,12 @@ spec:
           httpGet:
             path: /_healthz
             port: http
-          initialDelaySeconds: 5
-          periodSeconds: 5
-          timeoutSeconds: 5
-          successThreshold: 1
-          failureThreshold: 3
+{{ toYaml .Values.deployment.artifacts.readinessProbe | indent 10 }}
```

```suggestion
{{ toYaml .Values.deployment.artifacts.readinessProbe | indent 4 }}
```

Is not it better ? If it's 10, we will have a freaking big yaml file. I let you choose the solution you prefer

---

It's not about being better, it's about being accurate. If I set it to `4` it will break the code.